### ISSUE TITLE
"Configuring ISHCM web UI button bars" tutorial contains incorrect reference towards ISHType values

### ISSUE THREAD

**Issue by snijderr** [sdl/ISHDeploy#6]
The "Configure a bar" section within the "Configuring ISHCM web UI button bars" tutorial contains an issue with respect to the ISHType declaration which is used to restrict access to the button.

In the sample a similar hash as the one supplied below is used.

```powershell
$hash=@{
    Name="Custom1"
    Icon="Custom/Images/custom1.png"
    JSFunction="namespace.custom1"
    ISHType=ISHLibrary
}
```

However if I try to execute this I get the following error message returned in the Powershell window:
```powershell

ISHLibrary : The term 'ISHLibrary' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path wa
s included, verify that the path is correct and try again.
At C:\LcaCD\NYR.PATCH.Infoshare.12.0.0.AlfaLaval-PDMIntegration\CustomerSpecificFiles\FilesToExec\DeployScripts\Enable-PDM-Integration.ps1:42 char:10
+     ISHType=ISHLibrary
+             ~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ISHLibrary:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

If I update the hash inorder to have the ISHLibrary statement as an string it works.

```powershell
$hash=@{
    Name="Custom1"
    Icon="Custom/Images/custom1.png"
    JSFunction="namespace.custom1"
    ISHType="ISHLibrary"
}
```

Can this be corrected in the documentation so that others do not make the same mistake when providing ISHType values to restrict the access towards the added button.

**Comment by Sarafian** [comment]
@snijderr As I can't see the difference between the hash differences, I've compared them to verify they are the same. At this point I don't understand what the issue. Requesting clarification.

**Comment by snijderr** [comment]
Hi Alex,

I did make a copy paste error. The first hash should not have the quote characters around the ISHLibrary string. I have updated this in the original issue description.

This is the same syntax as is provided in the documentation samples on the tutorial pages.

Sorry for this confusion.

Raf

**Comment by Sarafian** [comment]
@snijderr this is not an ISHDeploy error but one related to PowerShell syntax.
When using splatting parameters you create a hash. As such, the hash values need to be in quotes when representing a string, even if the string will convert to an enum later on. Only by specifying the full .NET type name you can avoid the quotes. 

When not using splatting, the parameter value, even when it is just string, can be provided without double quotes.

This is an example
```powershell
$hashWorks=@{
   Name="ISHDeploy.12.0.3"
}
Find-Module @hashWorks
Find-Module -Name ISHDeploy.12.0.3
```

when executing 
```powershell
$hash=@{
Name=ISHDeploy.12.0.3
}
```
The powershell throws an error as it doesn't know what `ISHDeploy.12.0.3` is. This is something specific to PowerShell syntax. I'll close the issue.

**Comment by snijderr** [comment]
So the documentation will be changed to overcome this Powershell error (as it is a know not working structure by us)?
See the screenshot below of the tutorial content explaining our customers/partners how to use the cmdlets.

![image](https://cloud.githubusercontent.com/assets/1891044/24744130/30fcfe1a-1ab1-11e7-9869-e2741ad85b89.png)

**Comment by Sarafian** [comment]
Indeed. My apologies. I didn't see the part.

**Comment by Sarafian** [comment]
Fixed scheduled for the next release!

**Comment by Sarafian** [comment]
Fixed with new release 1.3
